### PR TITLE
[Cost 2005] - Invalidate Redis cache for all sources when currency is changed

### PR DESCRIPTION
**Pull request by ddonahue007**
closes: https://issues.redhat.com/browse/COST-2005

Fixes caching invalidation with currency changes in settings page
Fixes caching for invalidation cost saving changes in settings page

### CODE REVIEW

**Review by adberglund — APPROVED** [review]

File: koku/koku/cache.py [hunk 1]
```diff
@@ -89,3 +89,22 @@ def invalidate_view_cache_for_tenant_and_source_type(schema_name, source_type):
 
     for cache_key_prefix in cache_key_prefixes:
         invalidate_view_cache_for_tenant_and_cache_key(schema_name, cache_key_prefix)
+
+
+def invalidate_view_cache_for_tenant_and_source_types(schema_name, source_types):
```
[the remark this answers is not in the document]
Comment (ddonahue007) [reply koku/koku/cache.py:94]: Yes talking with @adberglund this is something that we see be using in the near future, there is test coverage included, so we are covered from that aspect.

**Comment by shannon-donahue** [comment]
lgtm

**Review by shannon-donahue — APPROVED** [review]
lgtm

**Review by dlabrecq — APPROVED** [review]

**Pull request merged**